apps/rehabstod/src/pages/Welcome/Welcome.tsx

```diff
@@ -9,7 +9,24 @@ export function Welcome() {
   const { selectedLogin, selectedUnit, freeText, selectedFilter } = useAppSelector((state) => state.welcome)
   const dispatch = useAppDispatch()
   const [triggerDefaultTestDataQuery, { isLoading: testDataLoading, data: response }] = useCreateDefaultTestDataMutation()
+  const [triggerCreateSickLeave, { isLoading: createSickLeaveLoading, data: certificateId }] = useCreateSickLeaveMutation()
+  const { data: testDataOptions, isLoading: testDataOptionsLoading } = useGetTestDataOptionsQuery()
   const { isLoading, fakeLogins } = useWelcome()
+  const [careProviderId, setCareProviderId] = useState('TSTNMT2321000156-ALFA')
```

Comment:
Nice, approved! I think this could have been made into a test data object and the default values could then have been placed in a WelcomeSlice or similar. But since this is for test purposes this works too :)